proposals/0007-access-policy.md

```diff
@@ -0,0 +1,157 @@
+---
+simd: '0007'
+title: SIMD Access Policy
+authors:
+- Jacob Creech (Solana Foundation)
+category: Meta
+type: Meta
+status: Draft
+created: 2022-12-10
+---
+
+## Summary
+
+3 levels of access, in the order of increasing access:
+
+1. Triage
+
+    Requirement: One voucher from anyone with level 2 or above access.
+
+2. Write
+
+    Requirement: One voucher from anyone with level 3 access or two from
+anyone with level 2 access. If only vouched by those with level 2 access, the
+user must have Triage access to be promoted.
+
+3. Maintain
+
+    Requirement: One voucher from anyone with level 3 access. This permission
+is usually reserved for those maintaining the SIMD process.
+
+Each of the levels implies having the previous levels - e.g. level 2 implies
+level 1. A contributor with Write access will be able to triage pull requests
+in accordance with Github access policies.
+
+## Motivation
+
+Shamelessly borrowing from [Mozilla's access
```

Comment:
love the honesty 😂

Reply:
@hydrogenbond007 please don't make comments that do not contribute to the discussion on SIMDs. It is not the place for them.